code/game/objects/items/storage/belt.dm

```diff
@@ -880,6 +880,34 @@
 	new /obj/item/ammo_magazine/revolver/mateba(src)
 	INVOKE_ASYNC(src, .proc/handle_item_insertion, new_gun)
 
+/obj/item/storage/belt/gun/rpg
+	name = "\improper TGMC rocket bag"
+	desc = "This backpack can hold 5 67mm shells or 80mm rockets. Somehow, with your magical hands, you can put the recoilless rifle in the backpack."
+	icon = 'icons/obj/items/storage/storage.dmi'
+	icon_state = "marine_rocket"
+	item_state = "marine_rocket"
+	var/base_icon = "marine_rocket"
```

Comment:
It's on a bottom type, you don't need this.

Reply:
It's needed else update_icon proc dies.

Reply:
Bravemole got better solution than base_icon